### ISSUE TITLE
App exceptions for features [enhancement]

### ISSUE THREAD

**Issue by nbporter** [noah-nuebling/Mac-Mouse-Fix#1113]
## Description
There should be an option to turn certain features on or off depending on the focused app.

## Rationale
Some games, such as Minecraft, do not work well with smooth or reversed scrolling. 

## Additional info
–

**Comment by motormayhem** [comment]
Yes I would like this as well I put in a similar request https://github.com/noah-nuebling/mac-mouse-fix/issues/1115
